src/dom.js

```diff
@@ -544,3 +544,18 @@ export function escapeCssSelectorIdent(win, ident) {
   // Polyfill.
   return cssEscape(ident);
 }
+
+
+/**
+ * Returns a frame element if available.
+ * @param {!Window} win
+ * @return {?HTMLIFrameElement}
+ */
+export function getFrameElement(win) {
+  try {
+    return win.frameElement;
+  } catch (e) {
```

Comment:
Could you clarify? When we call this method, we already know for certain that the window belongs to a friendly iframe. How else can we protect against an error here?